c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -1192,6 +1541,9 @@ class DeviceCachingAllocator {
 
   void* getBaseAllocation(Block* block, size_t* outSize) {
     std::lock_guard<std::recursive_mutex> lock(mutex);
+    TORCH_CHECK(
+        !block->expandable_segment_,
```

Comment:
I thought the peer access is for IPC ? or am i misunderstanding?

Reply:
peer access (supported) is for two devices to communicate within the same process (e.g. a `.to` kernel that specifies a different device). IPC is for two different processes.